data_model/device_types/section_Application-Device-Types.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
```

Comment:
Shouldn't these types of files be removed? There is a few so only commented on the first one in the list. 😄

Reply:
Can we have the scraper ignore spec documents that starts with "section_"?

Reply:
yup, fair - done.